### ISSUE TITLE
Image alternative text missing information - Leadership page

### ISSUE THREAD

**Issue by shawna-slh** [w3c/w3c-website#298]
**Describe the issue**
Image alternative text missing information.

**URL**
image: https://www.w3.org/comm/assets/graphics/w3c-le-governance.png
page: https://beta.w3.org/about/leadership/

**Recommended solution**
All information in the image needs to be provided to users who cannot see the image. Usually all text and diagram information in an image needs to be provided. It can be in alt text. Or it can be in the main text of the page. Or it can be in a separate page.

**Comment by koalie** [comment]
Thanks, @shawna-slh !
Done.
Updated alt text to: 
> Diagram explaining the standards development process on one side, and the reporting structure on the right. Web standards work is driven by W3C Staff, the community, W3C work groups and the W3C Director. The W3C TAG makes architecture choices that impact Web standards. The W3C AB updates the W3C Process with help from the community. The reporting structure is layered so that W3C Staff reports to W3C Management which reports to the Board of Directors in charge of finance, legal, HR matters and long-term strategic transactions.
